ArbGasInfo.sol

```diff
@@ -50,6 +50,7 @@ interface ArbGasInfo {
 
     /// @notice Get the gas accounting parameters. `gasPoolMax` is always zero, as the exponential pricing model has no such notion.
     /// @return (speedLimitPerSecond, gasPoolMax, maxBlockGasLimit)
```

Comment:
What about the other two values `gasPoolMax, maxBlockGasLimit`, should we add methods for them?